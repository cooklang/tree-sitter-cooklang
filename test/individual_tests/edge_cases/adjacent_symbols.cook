@salt{}#pan{}~{5%min}