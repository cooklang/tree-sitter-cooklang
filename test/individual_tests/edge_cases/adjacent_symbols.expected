(recipe
  (step
    (ingredient
      name: (ingredient_text))
    (cookware
      name: (cookware_text))
    (timer
      (amount
        (quantity)
        (units)))))